src/PowerShellEditorServices/Session/InvocationEventQueue.cs

```diff
@@ -225,6 +225,11 @@ private void OnInvokerUnsubscribed(object sender, PSEventUnsubscribedEventArgs e
             CreateInvocationSubscriber();
         }
 
+        private void OnInvokerUnsubscribed(object sender, PSEventArgs e)
```

Comment:
I'm not quite sure if this represents a bug in PSStandard btw. Changing from PSStandard to the PS Core SDK forced me to add this override.

Reply:
Definitely a bug, I opened an issue PowerShell/PowerShellStandard#50